rest/route/github.go

```diff
@@ -326,6 +329,23 @@ func (gh *githubHookApi) createPRPatch(ctx context.Context, owner, repo, calledB
 	return gh.AddIntentForPR(pr, pr.User.GetLogin(), calledBy)
 }
 
+func (gh *githubHookApi) refreshPatchStatus(ctx context.Context, owner, repo string, prNumber int) error {
+	p, err := patch.FindLatestGithubPRPatch(owner, repo, prNumber)
+	if err != nil {
+		return errors.Wrap(err, "finding patch")
+	}
+	if p == nil {
+		return errors.Errorf("couldn't find patch for PR '%s/%s:%d'", owner, repo, prNumber)
+	}
+	if p.Version == "" {
+		return errors.Errorf("patch '%s' not finalized", p.Version)
+	}
+
+	job := units.NewGithubStatusRefreshJob(p)
```

Comment:
How much lag are we expecting there to be between the user commenting and them seeing the refresh?

Reply:
When I tested it, it was pretty instant! Although I wrote this as a job bc it seemed more consistent / reusable for different things, we're just running the job right away as opposed to queueing it so I wouldn't expect delay.